src/settings/layout/blocks/ui/block.ts

```diff
@@ -926,6 +979,17 @@ class TableModal extends BasicModal<TableItem> {
                     this.block.calculate = v;
                 });
             });
+        new Setting(el)
+            .setName("Do Not Add Property as CSS Class")
```

Comment:
I usually try to prefer making these things _perform_ the task rather than turning it off. Could you please refactor this setting to that?